pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -65,13 +65,32 @@ type APIBindingSpec struct {
 	// +kubebuilder:validation:Required
 	Reference ExportReference `json:"reference"`
 
-	// acceptedPermissionClaims records the permissions that are granted
-	// to the bound workspace.
-	// Access is granted on a GroupResource basis and can be filtered on objects by many different selectors.
+	// permissionClaims records the permissions that are granted to the bound workspace. Access is granted per
+	// GroupResource. Individual claims may be accepted or rejected.
+
 	// +optional
-	AcceptedPermissionClaims []PermissionClaim `json:"acceptedPermissionClaims,omitempty"`
+	PermissionClaims []AcceptablePermissionClaim `json:"permissionClaims,omitempty"`
```

Comment:
should we keep `AcceptedPermissionClaims` until next wipe in addition and make the controller rewrite it? Then drop the old field in v0.9 ?

Reply:
Sure

Reply:
As discussed in Slack, proceeding w/the breaking change due to upcoming etcd wipe in 0.8